__tests__/app.test.js

```diff
@@ -111,6 +114,42 @@ describe('/api/articles', () => {
 
             })
     });
+});
+
+describe('/api/articles/:article_id/comments', () => {
+    test('GET 200 /api/articles/1/comments', () => {
+        return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then(({ body: { comments } }) => {
+                expect(comments.length).toBe(11)
+                comments.forEach((comment) => {
+                    expect(typeof comment.comment_id).toBe('number')
+                    expect(typeof comment.votes).toBe('number')
+                    expect(typeof comment.created_at).toBe('string')
+                    expect(typeof comment.author).toBe('string')
+                    expect(typeof comment.body).toBe('string')
+                    expect(typeof comment.comment_id).toBe('number')
+                })
+                expect(comments).toBeSorted({ key: 'created_at', descending: true })
+            })
+    });
 
+    test('GET 404 /api/articles/69/comments', () => {
+        return request(app)
+            .get('/api/articles/69/comments')
+            .expect(404)
+            .then(({ body: { msg } }) => {
+                expect(msg).toBe('Not found')
+            })
+    })
+    test('GET 400 /api/articles/cat/comments', () => {
+        return request(app)
+            .get('/api/articles/cat/comments')
+            .expect(400)
+            .then(({ body: { msg } }) => {
+                expect(msg).toBe('Invalid input')
+            })
+    })
 
```

Comment:
What about an article with no comments? What should be sent back? (i.e. it's not an error to not have comments!)